packages/storybook/src/nimble/table-column/base/table-column.stories.ts

```diff
@@ -136,6 +136,96 @@ const metadata: Meta<SharedTableArgs> = {
 
 export default metadata;
 
+interface ApiTableArgs extends SharedTableArgs {
+    content: undefined;
+    columnHidden: undefined;
+    columnId: undefined;
+    actionMenuSlot: undefined;
+    actionMenuLabel: undefined;
+    sortDirection: undefined;
+    sortIndex: undefined;
+    groupIndex: undefined;
+    groupingDisabled: undefined;
+    fractionalWidth: undefined;
+    minPixelWidth: undefined;
+}
+
+export const api: StoryObj<ApiTableArgs> = {
+    parameters: {
+        // Story used by documentation, not needed for visual comparison.
+        chromatic: { disableSnapshot: true }
+    },
+    argTypes: {
+        content: {
+            name: 'default',
+            description: 'Text or an icon added to the default slot of the column will be placed in the column header. See **Setting header content** for more information.',
```

Comment:
I would suggest we link this and other similar references to the section they're referencing (i.e. `#setting-header-content`). Unfortunately, I can't get links to open in the same iframe. They instead load the iframe content as the top-level page, losing the navigation pane, etc. Seems like the default `target` value for a link should be `"_self"`, but it doesn't work that way. If I manually add `target="_self"` to the generated link in dev tools, then it works properly.

Reply:
Yeah, Milan and I played with this before and came to the same conclusion. I've documented the best practice of marking headings as bold in the storybook CONTRIBUTING.